spec/person_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../classes/person'
+
+describe Person do
```

Comment:
- [ Required ] Please consider adding a unit test for the `off_age`, `add_rental`, `correct_name`, and `to_json` methods in your class `Person`, the project requirements state that all methods in your classes need to be tested.

## Project requirements
![Screenshot from 2023-03-30 23-55-19](https://user-images.githubusercontent.com/16392046/228962573-8389fc09-4ce6-42de-b1af-aefd25f2d036.png)